recipes/libbacktrace/all/conanfile.py

```diff
@@ -81,8 +77,8 @@ def generate(self):
 
         if is_msvc(self):
             env = Environment()
-            compile_wrapper = unix_path(self, self._user_info_build["automake"].compile)
-            ar_wrapper = unix_path(self, self._user_info_build["automake"].ar_lib)
+            compile_wrapper = unix_path(self, self.conf.get("user.automake:compile-wrapper"))
+            ar_wrapper = unix_path(self, self.conf.get("user.automake:lib-wrapper"))
```

Comment:
Are these variables required? Should we raise an error if they are None? I mean, something like this:

```python
    def validate(self):
        assert self.conf.get("user.automake:compile-wrapper"), "Please define xxxx"
        assert self.conf.get("user.automake:lib-wrapper"), "Please define xxxxxx"
```
It would be great if the `conf` would have a new parameter like `required=True`... 🤔

Reply:
Not mandatory, it will use regular compiler in case not filled.